posts/2022-05-05-real-time-vs-analytical-data.md

```diff
@@ -0,0 +1,100 @@
+---
+title: 'Types of Data: Real-time data vs. Analytical Data'
```

Comment:
Types of Data: Real-time Data vs. Analytical Data

Reply:
Fixed in latest push.